sdk/src/types/block/payload/mod.rs

```diff
@@ -80,6 +80,17 @@ impl Payload {
     crate::def_is_as_opt!(Payload: SignedTransaction, TaggedData);
 }
 
+impl WorkScore for Payload {
+    fn work_score(&self, params: WorkScoreParameters) -> u32 {
+        // 1 byte for the payload kind
+        1 + match self {
```

Comment:
What's that 1+ for ?

Reply:
the payload kind has to be taken into account ... not just the packed byte length, so it's +1

Reply:
Yeah Ok but I was asking because it wasn't grouped with packed_len

Reply:
you're right ... already fixed